### DISCUSSION TITLE
Ory Kratos Python SDK example

### DISCUSSION THREAD

**Discussion by Ankit-Mandloi47b** [ory/awesome-ory#79] [Q&A]
i have a query regarding integration of ory kratos with my gui (other than the self service gui image) using ory python sdk  in self hosting environment, i am stuck on implementing it .Can you please help me or provide any reference,it would be a great help!

**Comment by vinckr** [comment]
Hello @Ankit-Mandloi47b 

Have you seen https://github.com/ory/examples/tree/master/django-ory-cloud ? 

Would be great to have more Python examples, open for contributions!
